### ISSUE TITLE
Cast Picture Question

### ISSUE THREAD

**Issue by r3arsh** [PAhelper/PhoenixAdult.bundle#1779]
So I've noticed something kind of weird with the cast pictures across my different libraries.

Let's say I have a library for Vixen and another library for Brazzers.

I match everything up for Vixen and the cast photos come up fine. Then I start matching everything up for Brazzers and the cast photos come up fine.

When I go back to Vixen, now some cast photos are missing.

What I'm trying to figure out is the cast photos being shared and updated between the sites?

So this scenario:
1. Actress Riley has a scene in Vixen and I match up the scene and the agent is able to pull the cast picture.
2. Then, I have a scene with Riley in Brazzers but she doesn't have a cast photo or the agent isn't able to the pull the cast photo for some reason so it ends up being blank.
3. Then, I go back to Riley's scene in Vixen and her cast photo is gone.

Does this confirm the agent is sharing the cast photos between libraries? It may seem that way but just wanted confirmation from the developers to see that's how it's working.

If this is the case, how does the agent handle it when there are two actress's with the same name?

Sounds confusing, I know but any feedback would be appreciated.

Thanks.

**Comment by captaincoitus** [comment]
> Does this confirm the agent is sharing the cast photos between libraries? It may seem that way but just wanted confirmation from the developers to see that's how it's working.
Yes, the cast pictures are shared across libraries. However, to the best of my knowledge, once there is a cast picture in the database, the picture coming up in the newer searches is discarded. Maybe I am wrong about this, I don't have an extensive background in python. I am a Full Stack Dev specializing in Java and Frontend.
